code/residential-parking-permits/rpp.css

```diff
@@ -89,9 +175,787 @@ a.big-button {
+#kn-report-view_1595-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(29){fill: #f45b5b;}
+#kn-report-view_1595-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(30){fill: #91e8e1;}
+
+/*Green Zone*/
+/*Current Year*/
+/*Active Permits by Type*/
+#kn-report-view_1335-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(1){fill: #7cb5ec;}
+#kn-report-view_1335-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(2){fill: #434348;}
+#kn-report-view_1335-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(3){fill: #90ed7d;}
+
+/*Total Permits Issued by Type*/
+#kn-report-view_1342-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(1){fill: #7cb5ec;}
+#kn-report-view_1342-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(2){fill: #434348;}
+#kn-report-view_1342-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(3){fill: #90ed7d;}
+
+/*Completed Applications by Type*/
+#kn-report-view_1349-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(1){fill: #7cb5ec;}
+#kn-report-view_1349-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(2){fill: #434348;}
+#kn-report-view_1349-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(3){fill: #90ed7d;}
+#kn-report-view_1349-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(4){fill: #f7a35c;}
+
+/*Completed Applications by Payment Type*/
+#kn-report-view_1356-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(1){fill: #7cb5ec;}
+#kn-report-view_1356-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(2){fill: #434348;}
+#kn-report-view_1356-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(3){fill: #90ed7d;}
+#kn-report-view_1356-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(4){fill: #f7a35c;}
+#kn-report-view_1356-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(5){fill: #8085e9;}
+#kn-report-view_1356-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(6){fill: #f15c80;}
+#kn-report-view_1356-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(7){fill: #e4d354;}
+
+/*Completed Applications by Transaction Amount - we repeat the HighChart colors for up to a possible 27+ different Transaction Amounts*/
+#kn-report-view_1594-1 div.highcharts-container svg g.highcharts-series-group g.highcharts-series rect:nth-child(1){fill: #7cb5ec;}
```

Comment:
so many colors! there's also the option for the future (or if the transaction amounts are more than 30 rows) to do something like `:nth-child(10n+1)`, `:nth-child(10n+2)`
https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child#example_selectors

Reply:
oh thats true since there are only 10 colors and we could limit the code to 20 lines instead of 27 and we wouldnt have to update it if it became more than 27.

Reply:
yeah, if there are only 10 colors, you only have to do 10n+1 through 10n+10

Reply:
wouldnt it be 10n, 10n+1 through 10n+9 ?